pkg/discovery/queries.go

```diff
@@ -257,7 +257,12 @@ func filterResources(gvrs map[schema.GroupVersion][]metav1.APIResource, namespac
 }
 
 // QueryPodLogs gets the pod logs for each pod in the given namespace.
-func QueryPodLogs(kubeClient kubernetes.Interface, recorder *QueryRecorder, ns string, cfg *config.Config) error {
+// If namespace is not provided, get pod logs from field selectors
```

Comment:
I'd end this with a period `.`; it doesn't seem to flow into the next line as a sentence.

tiny nit: I think it reads better to say "pod logs using field selectors" instead of "from field selectors"